### PR TITLE
[DROOLS-3571] Rename Test Scenario

### PR DESCRIPTION
**Pull request by annabaker**
@danielezonca

### CODE REVIEW

**Comment by danielezonca** [comment]
jenkins please retest this

**Comment by Rikkola** [comment]
There is one more String in kie-wb-common related to editor permissions.

**Comment by Rikkola** [comment]
https://github.com/kiegroup/kie-wb-common/blob/dbbd1f608d3c3cf47cfa4cc99a1695a693c1fed9/kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/resources/org/kie/workbench/common/workbench/client/resources/i18n/DefaultWorkbenchConstants.properties#L88

Here is the kie-wb-common one.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties [hunk 1]
```diff
@@ -13,13 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-newScenarioSimulationDescription=Test Scenario (Preview)
+newScenarioSimulationDescription=Test Scenario
```
Comment (kkufova) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: There is a redundant space after the name.

> Test Scenario_

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
 #
-newScenarioSimulationDescription=Test Scenario (Preview)
+newScenarioSimulationDescription=Test Scenario 
 factColumnHeader=Fact
 fieldColumnHeader=Field
 contextColumnHeader=Context
-addScenarioSimulation=Add Test Scenario (Preview)
-scenarioSimulationEditorTitle=Test Scenarios (Preview)
-scenarioSimulationResourceTypeDescription=Test Scenarios (Preview)
+addScenarioSimulation=Add Test Scenario
```
Comment (kkufova) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: Redundant space.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties [hunk 3]
```diff
@@ -13,13 +13,13 @@
...
-newScenarioSimulationDescription=Test Scenario (Preview)
+newScenarioSimulationDescription=Test Scenario 
 factColumnHeader=Fact
 fieldColumnHeader=Field
 contextColumnHeader=Context
-addScenarioSimulation=Add Test Scenario (Preview)
-scenarioSimulationEditorTitle=Test Scenarios (Preview)
-scenarioSimulationResourceTypeDescription=Test Scenarios (Preview)
+addScenarioSimulation=Add Test Scenario 
+scenarioSimulationEditorTitle=Test Scenarios
```
Comment (kkufova) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: Redundant space.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties [hunk 4]
```diff
@@ -13,13 +13,13 @@
...
+newScenarioSimulationDescription=Test Scenario 
 factColumnHeader=Fact
 fieldColumnHeader=Field
 contextColumnHeader=Context
-addScenarioSimulation=Add Test Scenario (Preview)
-scenarioSimulationEditorTitle=Test Scenarios (Preview)
-scenarioSimulationResourceTypeDescription=Test Scenarios (Preview)
+addScenarioSimulation=Add Test Scenario 
+scenarioSimulationEditorTitle=Test Scenarios 
+scenarioSimulationResourceTypeDescription=Test Scenarios
```
Comment (kkufova) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: Redundant space.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties (hunk 1 above)
Comment (annabaker) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: fixed

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties (hunk 2 above)
Comment (annabaker) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: fixed

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties (hunk 3 above)
Comment (annabaker) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: fixed

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties (hunk 4 above)
Comment (annabaker) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: fixed

**Comment by annabaker** [comment]
> https://github.com/kiegroup/kie-wb-common/blob/dbbd1f608d3c3cf47cfa4cc99a1695a693c1fed9/kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/resources/org/kie/workbench/common/workbench/client/resources/i18n/DefaultWorkbenchConstants.properties#L88
> 
> Here is the kie-wb-common one.

https://github.com/kiegroup/kie-wb-common/pull/2445

**Comment by danielezonca** [comment]
@manstis 

Can you please squash and merge?

**Pull request merged**